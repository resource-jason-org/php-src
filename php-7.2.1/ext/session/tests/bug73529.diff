006+ bool(true)
006- bool(false)