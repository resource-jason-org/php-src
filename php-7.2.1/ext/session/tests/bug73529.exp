bool(true)
array(1) {
  ["foo"]=>
  string(3) "bar"
}
bool(false)
array(0) {
}