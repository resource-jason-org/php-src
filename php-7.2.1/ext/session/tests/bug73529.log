
---- EXPECTED OUTPUT
bool(true)
array(1) {
  ["foo"]=>
  string(3) "bar"
}
bool(false)
array(0) {
}
---- ACTUAL OUTPUT
bool(true)
array(1) {
  ["foo"]=>
  string(3) "bar"
}
bool(true)
array(0) {
}
---- FAILED
