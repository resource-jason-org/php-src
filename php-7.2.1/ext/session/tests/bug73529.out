bool(true)
array(1) {
  ["foo"]=>
  string(3) "bar"
}
bool(true)
array(0) {
}